src/js/controllers/ctrl-widget-settings.js

```diff
@@ -50,35 +50,46 @@ app.controller('WidgetSettingsController', function(
 	}
 
 	const _toggleNormalAccess = () => {
-		if (($scope.guestAccess = true)) {
-			$scope.guestAccess = false
-		}
-		if (($scope.embeddedOnly = true)) {
+		// disallow student made widgets from being toggled out of guest mode
```

Comment:
These 3 toggle functions need some work in the future.

These aren't toggling type actions, they should act like radio buttons.

Right now thinking about switching them to use ng-model and add an onChange to that value to run through the combined rules of all 3 of these toggle methods we have now.